pkg/client/integration.go

```diff
@@ -21,21 +21,64 @@ func (c *Client) AddIntegration(config *v1alpha1.K8sGPT) error {
 		return err
 	}
 
-	if resp.Trivy.Enabled == config.Spec.Integrations.Trivy.Enabled {
-		fmt.Println("Skipping trivy installation, already enabled")
+	skipTrivy := false
+	skipKyverno := false
+
+	if resp.Trivy.Enabled {
+		if config.Spec.Integrations.Trivy != nil {
+			if config.Spec.Integrations.Trivy.Enabled {
+				fmt.Println("Skipping trivy installation, already enabled")
+				skipTrivy = true
+			}
+		}
+	} else {
```

Comment:
I am not sure if we need this else at all

Reply:
tbh, it needs a refactor. The trivy integration check is a horrid hack back from when it was the only one around

Reply:
Agree, I thought a bit about it, but since I wasn't involved with prior work I tried to do something similar / simple. But agree, could use some redo.